tests/product/tests/conftest.py

```diff
@@ -217,3 +222,17 @@ def pytest_generate_tests(metafunc):
             params.ids = params.ids[int(test_range_start) :]
 
     metafunc.parametrize(params.argnames, params.argvalues, ids=params.ids)
+
+
+def parametrize_eks_params(func_details):
+    """
+    This function creates parametrization for EKS test cases
+    @param func_details: metafunc
+    @return:
+    """
+
```

Comment:
PEP 257: [There's no blank line either before or after the docstring.](https://peps.python.org/pep-0257/)

Reply:
removed